flytekit/common/tasks/notebook.py

```diff
@@ -0,0 +1,542 @@
+from __future__ import absolute_import
```

Comment:
can we put this in `flytekit.contrib.notebooks` or similar as an example of a 'contrib'?  Also, that can buy us some flexibility in terms of the simplification work I want to take on to make this easier to accomplish since it won't have the same guarantees of our 1st class tasks. Once stabilized, we can upstream into the 1st class package. @kumare3 what do you think?

Reply:
moved